CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 1.6.1 (Upcoming)
 
+### FEATURES:
+* builder/vsphere-clone: Add `boot_command` support to vsphere-clone builder,
```

Comment:
I think this is a better fit for the `improvements` section.

Reply:
Moved from FEATURES to IMPROVEMENTS